class.jetpack-jitm.php

```diff
@@ -15,9 +15,20 @@ class Jetpack_JITM {
 	/**
 	 * Initializes the class, or returns the singleton
 	 *
-	 * @return Jetpack_JITM
+	 * @return Jetpack_JITM | false
 	 */
 	static function init() {
+		/**
+		 * Filter to turn off all just in time messages
+		 *
+		 * @since 3.7.0
+		 *
+		 * @param bool true Whether to show just in time messages.
```

Comment:
This docblock statement doesn't seem to match with what's actually going on. We are actually passing `false` to the filter. Also, the docblock should probably read, "Whether to turn off just in time messages" to match the logic.

Reply:
It looks like we do hook on the filter and return true in `class.jetpack.php`. It seems like we should just pass true here and remove our filter from `class.jetpack.php` perhaps?